packages/icon-memory/src/components/MatchedCards.vue

```diff
@@ -1,14 +1,15 @@
 <script setup lang="ts">
   import IconCard from './IconCard.vue';
-  import type { IconCardProps } from './IconCard.vue';
+  import type { Card } from './IconMemory.vue';
```

Comment:
We don't need all the props from the IconCard here, we just want the basic Card type.